recipes/dlatk/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "dlatk" %}
+{% set version = "1.2.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dlatk-{{ version }}.tar.gz
+  sha256: fc4c8f12c597623facb5100e6d836a191459655938c83ccc352df377bd07cebd
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python >=3.6
+  run:
+    - mysqlclient
+    - nltk >=3.2.1
+    - numpy
+    - pandas >=0.20.3
+    - patsy >=0.2.1
+    - python >=3.6
+    - python-dateutil >=2.5.0
+    - scikit-learn
+    - scipy >=0.13.3
+    - sqlalchemy >=0.9.9
+    - statsmodels >=0.5.0
+
+test:
+  imports:
+    - dlatk
+    - dlatk.lexicainterface
+  requires:
+    - pip
+
+about:
+  home: http://dlatk.wwbp.org
+  summary: DLATK is an end to end human text analysis package, specifically suited for social media and social scientific applications. It is written in Python 3 and developed by the World Well-Being Project at the University of Pennsylvania and Stony Brook University.
```

Comment:
The summary should be short. For a longer bit of text you should use the `description` field as well.

Reply:
fixed in https://github.com/conda-forge/staged-recipes/pull/14068/commits/4a1d6b14270605e94cb55ce8f9633a6b45ed9869